specs-go/v1/config.go

```diff
@@ -75,6 +75,7 @@ type History struct {
 }
 
 // Image is the JSON structure which describes some basic information about the image.
+// This provides the `application/vnd.oci.image.config.v1+json` mediatype when marshalled to JSON.
```

Comment:
> `oci-cas get some-image "${DIGEST}" | some-tool --add-option --media-type "${MEDIA_TYPE}"`

IMO that's ridiculous. You now need to carry around the media type of an object separate to the object. Why? What is the benefit? Why not just follow along with literally every other format that exists and make it self-describing? Why has this discussion been going on for so long?